superset/models/dashboard.py

```diff
@@ -382,7 +382,7 @@ def export_dashboards(  # pylint: disable=too-many-locals
     ) -> str:
         copied_dashboards = []
         datasource_ids = set()
-        for dashboard_id in dashboard_ids:
+        for dashboard_id in set(dashboard_ids):
```

Comment:
If there's an issue with duplicates—where order doesn't matter—is there merit in changing the `export_dashboards` method to have `dashboard_ids` as `set[int]`?

Reply:
Good point, let me fix it.